egs2/chime6/asr1/run.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+
+
+train_set=train_worn_simu_u400k_cleaned
+valid_set=dev_gss_multiarray
+test_sets="dev_gss_multiarray"
+
+
+asr_config="conf/tuning/train_asr_transformer_wavlm_lr1e-3_specaug_accum1_preenc128_warmup20k.yaml"
+inference_config="conf/decode_asr_transformer.yaml"
+lm_config="conf/train_lm.yaml"
+
+
+use_lm=false
+use_word_lm=false
+word_vocab_size=65000
+
+./asr.sh \
+    --lang en \
+    --bpe_nlsyms "$(< data/nlsyms.txt tr '\n' ',' | sed 's/,$//g')" \
```

Comment:
Does `data/nlsyms.txt` exist before stage 1? Will this throw an error if the file does not exist?

Reply:
Thanks for pointing out this.